test/JDBC/expected/Test_parallel_query.out

```diff
@@ -0,0 +1,223 @@
+-- tsql
+CREATE LOGIN l1 WITH PASSWORD = '12345678'
+GO
+
+CREATE USER l1
```

Comment:
we should have a test where
accessing temp table inside parallel worker should give error
accessing temp table as outer scan should not give an error
accessing PG temp table inside parallel worker should give error
accessing PG temp table as outer scan should not give an error (but this will not skip the check, rather check will pass)
accessing PG temp table as outer scan should give an error (due to perm issue)
accessing PG regular table in both case should always give an error
mix of all three tables in same query

Reply:
Added